scripts/travis/build.sh

```diff
@@ -53,9 +53,29 @@ fi
 ./gradlew build $EXTRA_ARGS
 EXIT_CODE=$?
 
+if [ $EXIT_CODE != 0 ]; then
+  # Kill the waiter job
+  echo "[Ping] Killing the waiter job"
+  kill $WAITER_PID
+
+  echo "Java 8 build and test failed"
+  exit 1
+fi
+
+# Run tests with Java 11
+# .travis.yml provides this environment variable
+EXIT_CODE_JAVA11=0
+if [ ! -z "$JAVA11_HOME" ]; then
+  echo "Running tests with Java 11"
+  ./gradlew check -PalternateJvm=$JAVA11_HOME/bin/java
```

Comment:
We don't run tests in all cases. Note how line 53 has `$EXTRA_ARGS`, which can sometimes be `-x test`. This is when we're doing the release-and-publish job. We don't want to run java 11 tests in this case.

Reply:
Fixed